server/routes/sockets/room.js

```diff
@@ -1,13 +1,62 @@
 const _ = require('lodash');
+const Constants = require('../../constants');
 
 class Room {
-    constructor(ioRef, roomID) {
+    constructor(ioRef, roomID, roomConfig) {
         // This is the reference to the io server
         this.ioRef = ioRef;
         // The reference to the appropriate io room ("channel")
         this.roomID = roomID;
+        // The reference to the roomConfig
+        this.roomConfig = roomConfig;
+        // The reference to the room's deck
+        this.deck = this.roomConfig.deck;
```

Comment:
I don't think that is a risk just because there is a separation between the client, the database, and the sockets. Regardless, I'll turn it into a copy with a spread operator instead. :+1: